pds4indextools/pds4_create_xml_index.py

```diff
@@ -75,56 +154,80 @@ def default_value_for_nil(config, data_type, nil_value):
     return default
 
 
-def convert_header_to_tag(path, root, namespaces):
-    """Convert an XPath expression to an XML tag.
+def extract_logical_identifier(tree):
+    """
+    Extract the logical_identifier element from an XML tree.
 
-    Inputs:
-        path          XPath expression.
-        root          The root element of the XML document.
-        namespaces    Dictionary of XML namespace mappings.
+    Args:
+        tree (ElementTree.Element): The XML tree.
 
     Returns:
-        Converted XML tag.
+        str or None: The text content of the logical_identifier element,
+                     or None if not found.
```

Comment:
This is fine, but just FYI the normal indentation for multi-line Args or Returns is to wrap the text with a 4-space indent, like:

```
        str or None: The text content of the logical_identifier element,
            or None if not found.
```

That way you don't have to squeeze everything into a narrow column on the right.

Reply:
Oh! That actually helps quite a bit.